shopware/core/6.4/manifest.json

```diff
@@ -51,36 +51,13 @@
         ".env.local",
         ".env.local.php",
         ".env.*.local",
-        "custom/*",
-        "!custom/plugins/.gitkeep",
-        "!custom/static-plugins/",
-        "/public/fonts/*",
-        "!/public/fonts/.gitkeep",
-        "/public/img/*",
-        "!/public/img/.gitkeep",
-        "/public/css/*",
-        "!/public/css/.gitkeep",
-        "/public/js/*",
-        "!/public/js/.gitkeep",
-        "/public/media/*",
-        "!/public/media/.gitkeep",
-        "/public/theme/*",
-        "!/public/theme/.gitkeep",
-        "/public/thumbnail/*",
-        "!/public/thumbnail/.gitkeep",
-        "/public/assets/*",
-        "/public/sitemap/*",
-        "!/public/sitemap/.gitkeep",
-        "/public/bundles/*",
```

Comment:
Is still public/bundles and so on completely ignored? 🤔 I don't see it

Reply:
You are right, they should be ignored, but there is no .gitkeep inside so I only added the public folders back:
```
/public/bundles/*
/public/media/*
/public/theme/*
/public/thumbnail/*
/public/sitemap/*

```

Reply:
sounds good 👍